workspace_status.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+#`p_` takes two arguments to define a bazel workspace status variable:
```

Comment:
backticks in shell scripts always make me nervous. :)

Reply:
Hmm, I should remove it... in a later patch, heh.